applications/tari_validator_node/src/grpc/services/base_node_client.rs

```diff
@@ -124,6 +124,37 @@ impl BaseNodeClient for GrpcBaseNodeClient {
             .collect()
     }
 
+    async fn get_contract_utxos(
```

Comment:
This is basically the same as `get_current_contract_outputs` above. I think it's better to refactor `get_current_contract_outputs` to return `UtxoMinedInfo`.